### ISSUE TITLE
Tooltips inside of dropdown menus don't wrap text

### ISSUE THREAD

**Issue by aaronfrost** [vmware/clarity#1617]
**Select one ...**  (check one with "x")
```
[X] bug
[ ] feature request
[ ] enhancement
```

### Expected behavior

Putting a tooltip inside of a dropdown menu item should not lose the default wrapping of text that normally exists in tooltips

### Actual behavior

When you put a tooltip inside of a dropdown menu, you lose the wordwrap, cause the menu forces `white-space: no-wrap` on it's children. You have to override it manually. 

### Reproduction of behavior

* Include a link to the reproduction scenario you created by forking one of the Clarity Plunker Templates:
https://embed.plnkr.co/eqwFPgdIfGVzpnmS5eYD/

In this example, you have a button with a tooltip that does wrap, and then in the dropdown, the menu item doesn't wrap, but is using the same tooltip text. 

### Environment details

* **Angular version:** 4.x.x

* **Clarity version:** 0.10.8

* **OS and version:** OSX 10.12.6

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

**Comment by adityarb88** [comment]
@aaronfrost: Thanks for reporting this bug and providing the plnkr to replicate this issue. We will add this our queue so that its fixed.

**Comment by aaronfrost** [comment]
No, thank you guys for building clarity and sharing it with us. You have made a lot of things easier. Seeing your patterns are more than interesting. 

I am one of the organizers of ng-conf. I think that your team should put together a few of the greatest patterns and innovations you have done, and prepare them to submit in a talk to ng-conf. The CFP is open right now. Some of the stuff you have done is pretty epic. Things like `*clrIfOpen` are things I haven't seen anyone else do. If you are interested, please submit and come be a part of ng-conf. I know that vmware was a sponsor last year. You should definitely submit to the CFP.

**Comment by youdz** [comment]
Hi Aaron, this is really good to hear, we're glad you like the patterns we're trying to push. We're spending a lot of time and effort on trying to be as innovative as possible with a very young Angular framework, were everything is up for grabs. 😄 

We did submit a few talks at ng-conf last year, a couple of them on the specific patterns we use to make our components easy to use and reusable, but they weren't accepted. At the conf, we realized that the Angular Material team had submitted very similar talks, so we understood the decision better.

We'll still submit papers this year, definitely. Thanks for the suggestion on what to submit, we'll prepare something along those lines. I guess we'll see you there!
